### PR TITLE
feat(ui): PrimaryButton

### PR DESCRIPTION
**Pull request by zielu92**

### CODE REVIEW

**Review by Pittawat2542 — CHANGES_REQUESTED** [review]
Please
1. Rebase from `main`
```
git pull --rebase origin main 
```
2. Ignore Jetbrain's IDE Setting files
3. Add screenshot
Thanks!

**Review by Pittawat2542 — CHANGES_REQUESTED** [review]
Please fix the following problems:
1. `.idea/` folder still showing up. :(
2. Be sure to rebase and force push
```
git pull --rebase origin main 
```
and then
```
git push -f origin PrimaryButton
```

**Comment by Pittawat2542** [comment]
> Thanks I fixed it already

Next time, don't forget to force push after rebase
```
git push -f origin PrimaryButton
```

**Review by Pittawat2542 — CHANGES_REQUESTED** [review]
I still seeing conflicts on `Index.tsx` and `package-lock.json` and `.idea/` folder still showing up. Please resolve these issues.

File: components/ui/PrimaryButton.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import styled from "@emotion/styled";
+
+const PrimaryButton = styled.button`
+  font-size: 1em;
```
Comment (Pittawat2542) [inline components/ui/PrimaryButton.tsx:4]: This does not need

File: components/ui/PrimaryButton.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  font-weight: bold;
+  margin: 1em;
```
Comment (Pittawat2542) [inline components/ui/PrimaryButton.tsx:6]: Don't use `em` with margin. Use `px` instead.

File: components/ui/PrimaryButton.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  padding: 0.4em 2em;
```
Comment (Pittawat2542) [inline components/ui/PrimaryButton.tsx:7]: Don't use `em` with margin. Use `px` instead.

**Review by Pittawat2542 — CHANGES_REQUESTED** [review]
`.idea/` folder still show up.

try add [this](https://www.toptal.com/developers/gitignore/api/webstorm) to `.gitignore`

image

**Comment by zielu92** [comment]
fixed styles

**Pull request merged**